apps/wallet/src/ui/app/pages/home/usdc-promo/useUsdcPromo.ts

```diff
@@ -8,6 +8,7 @@ type WalletUsdcPromo = {
 	promoBannerText: string;
 	promoBannerSheetTitle: string;
 	promoBannerSheetContent: string;
+	atcLabel: string;
```

Comment:
`ctaLabel` ?

Reply:
For this label

![Screenshot 2024-10-04 at 10 47 33 AM](https://github.com/user-attachments/assets/86932f4c-9edc-4760-91fb-e7381debbf33)

Reply:
I just mean the name, it's CTA not ATC ?

Reply:
o ya typo, let me change the text n contentful